platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/wiring/PlatformSchedulers.java

```diff
@@ -255,6 +257,12 @@ public static PlatformSchedulers create(
                         .withUnhandledTaskCapacity(config.issDetectorUnhandledCapacity())
                         .withMetricsBuilder(model.metricsBuilder().withUnhandledTaskMetricEnabled(true))
                         .build()
+                        .cast(),
+                model.schedulerBuilder("hashLogger")
+                        .withType(TaskSchedulerType.SEQUENTIAL_THREAD)
```

Comment:
I'd suggest making this a configurable value, similar to the other schedulers. Also, was there a reason why this is a `SEQUENTIAL_THREAD` scheduler instead of being a plain `SEQUENTIAL` scheduler? Absent a strong reason, `SEQUENTIAL` is almost always the better choice.

Reply:
@cody-littley It's logging to a file, so presumably IO bound, which calls for `SEQUENTIAL_THREAD`, right?

Reply:
i think log4j writes to a file on a separate thread. the reason why its a thread is because it used to be a thread, we don't want to change anything about the threading model at this point if we don't have to. changing such things should be done when we can adequately test it

Reply:
made it configurable.